src/main/java/TaskListener.java

```diff
@@ -0,0 +1,191 @@
+    }
+
+    private void printAddedTask(Task task) {
+        String firstSentence = "Got it. I've added this task:";
+        String tasksWord = taskList.getTasksCount() == 1 ? " task" : " tasks";
+        String secondSentence = "Now you have " + taskList.getTasksCount() + tasksWord + " in the list.";
+        printLines(firstSentence + System.lineSeparator() + task + System.lineSeparator() + secondSentence);
+    }
+
+    public static void greet() {
+        String logo =
+                        "   _____ __  ____________  __    ____  ________ __\n" +
+                        "  / ___// / / / ____/ __ \\/ /   / __ \\/ ____/ //_/\n" +
+                        "  \\__ \\/ /_/ / __/ / /_/ / /   / / / / /   / ,<   \n" +
+                        " ___/ / __  / /___/ _, _/ /___/ /_/ / /___/ /| |  \n" +
+                        "/____/_/ /_/_____/_/ |_/_____/\\____/\\____/_/ |_|  ";
+
+
+        printLines("Hello from", logo);
+
+        printLines("Hello! I'm SHERLOCK", "What can I do for you?");
+    }
+
+    private void addCommandHandler(String name) {
+        this.taskList.addTask(new Task(name, false));
+        printLines("added: " + name);
+    }
+
+    private void todoCommandHandler(String arguments) {
+        String name = arguments;
+        Todo todo = new Todo(name, false);
+        this.taskList.addTask(todo);
+        printAddedTask(todo);
+    }
+
+    private void deadlineCommandHandler(String arguments) {
+        {
+            int byCommandIndex = arguments.indexOf("/by");
+            if (byCommandIndex < 0) {
+                printLines("Please specify the /by command");
+                return;
+            }
+            String name = arguments.substring(0, byCommandIndex).trim();
+            String byArgumentValue = arguments.substring(byCommandIndex)
+                    .substring("/by".length()).trim();
+
+            if (name.isEmpty()) {
+                printLines("Please provide name argument");
+                return;
+            }
+
+            if (byArgumentValue.isEmpty()) {
+                printLines("Please provide /by argument");
+                return;
+            }
+            Deadline deadline = new Deadline(name, false, byArgumentValue);
+            this.taskList.addTask(deadline);
+            printAddedTask(deadline);
+        }
+    }
+
+    private void eventCommandHandler(String arguments) {
+        {
+            int fromCommandIndex = arguments.indexOf("/from");
+            int toCommandIndex = arguments.indexOf("/to");
+
+            if (fromCommandIndex < 0) {
+                printLines("Please specify the /from command");
+                return;
+            }
+            if (toCommandIndex < 0) {
+                printLines("Please specify the /to command");
+                return;
+            }
+
+            String name = arguments.substring(0, fromCommandIndex).trim();
+
+            String fromArgumentValue = arguments.substring(fromCommandIndex, toCommandIndex)
```

Comment:
I like how you avoid writing long lines of code by transferring some code to the next line! It really improves the readability of your code